src/Microsoft.Health.Fhir.Core/Features/Resources/Delete/FireEventForDeleteResourceBehavior.cs

```diff
@@ -0,0 +1,40 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using MediatR;
+using Microsoft.Health.Fhir.Core.Messages.Create;
+using Microsoft.Health.Fhir.Core.Messages.Delete;
+using Microsoft.Health.Fhir.Core.Messages.Upsert;
+
+namespace Microsoft.Health.Fhir.Core.Features.Resources.Delete
+{
+    public class FireEventForDeleteResourceBehavior :
```

Comment:
Maybe we should remove the event stuff for now since we're not using it. Same goes for the `UpsertedEvent`

Reply:
These should probably replace the AuditFilters...next time then.